### DISCUSSION TITLE
Liberate Auth !!!

### DISCUSSION THREAD

**Discussion by bezhanSalleh** [filamentphp/filament#295] [Ideas]
First of all, let me **thank you guys for building this amazing package**  👍.

**Current Auth:**
- not intuiative enough for new comers
- no customization options or reference in the docs
- to much unnessesary work to customize

**Proposed Auth:**
- scaffold auth assets i.e `php artisan make:filament-auth` or ` php artisan filament:auth`
- customize however you want
- upgrade the package without any concerns
- implement you own ...

I think it would be a good idea to make auth independent of the package which would lead the users to easily customize or implement their own solutions as they see fit.

Just an idea. Thanks again for the amazing package.

~ Bezhan

**Comment by ryangjchandler** [comment]
@bezhanSalleh Thanks for opening this discussion. We believe that the current auth setup is perfect for getting started quickly.

The steps required for installing into an existing app and/or using a custom authentication (`User` model) system are documented on the website too. If you're not understanding them, please let us know which parts are confusing and we'll try our best to improve them.

**Reply by bezhanSalleh** [reply]
hey @ryangjchandler I agree it is perfect. i only mean the login, reset password part of the authentication process and their views. cause right now until you dig into the package you don't know how to say remove the forgot password link or add another field.
